db/migrate/20150319212424_migrate_ncr_to_models.rb

```diff
@@ -2,19 +2,23 @@ class MigrateNcrToModels < ActiveRecord::Migration
   class TempProperty < ActiveRecord::Base
     self.table_name = 'properties'
   end
+
   class TempCart < ActiveRecord::Base
     self.table_name = 'carts'
     has_many :properties, -> { where hasproperties_type: 'Cart' },
-              foreign_key: "hasproperties_id"
+              foreign_key: "hasproperties_id",
+              class_name: 'TempProperty'
```

Comment:
my concern is more than it isn't failing than that you didn't know the rubocop style we're using. will have to look into that...